cosmwasm/contract.ts

```diff
@@ -121,24 +101,55 @@ const registerItsChain = async (
             },
         };
     });
+};
 
-    if (options.update) {
-        for (let i = 0; i < options.chains.length; i++) {
-            const chain = options.chains[i];
-            await validateItsChainChange(client, config, chain, chains[i]);
-        }
+const registerItsChain = async (
+    client: ClientManager,
+    config: ConfigManager,
+    options: ContractCommandOptions,
+    args?: string[],
+    fee?: string | StdFee,
+): Promise<void> => {
+    if (!args || args.length === 0) {
+        throw new Error('At least one chain is required');
+    }
+
+    const chains = buildItsHubChains(config, args);
+
+    const msg = [{ register_chains: { chains } }];
+
+    const chainsList = args.join(', ');
+    const defaultTitle = `Register ${chainsList} on ITS Hub`;
+    const defaultDescription = `Register ${chainsList} on ITS Hub`;
+
+    return executeContractMessage(client, config, options, 'InterchainTokenService', msg, fee, defaultTitle, defaultDescription);
```

Comment:
```suggestion
    const defaultTitle = `Register ${args.join(', ')} on ITS Hub`;

    return executeContractMessage(client, config, options, 'InterchainTokenService', msg, fee, defaultTitle);
```
explicitly reuse title for both and simplify, since that's what you're doing already